src/rulesets/oas/index.json

```diff
@@ -594,14 +595,15 @@
       "message": "{{error}}",
       "recommended": true,
       "severity": 0,
+      "formats": ["oas3"],
       "type": "validation",
-      "given": "$..parameters..[?(@.example && @.schema)]",
+      "given": "$..parameters..[?(@.schema && (@.example || @.examples))]",
```

Comment:
See the OAS3 spec for Parameter objects, they can taker example or examples at that top level, which is correct with what you've got here. 

<img width="870" alt="image" src="https://user-images.githubusercontent.com/67381/89888501-f5fa0700-dbc7-11ea-9b72-1a0ad0ee69c7.png">

They can also take the OAS `example` or the "JSON Schema" `examples`:

There can be an `example` inside the schema for a parameter or next to it.

<img width="895" alt="image" src="https://user-images.githubusercontent.com/67381/89888422-d6fb7500-dbc7-11ea-9783-f1e8b0f54f81.png">

The example key could be inside or next to schema, and it works the same. The examples key inside schema would be a "JSON Schema" examples which is a bare array (no values key), and the examples next to schema would be an OAS examples, which has the `values` key inside.